src/app/components/game-components/rectangular-game-component/RectangularGameComponent.ts

```diff
@@ -0,0 +1,23 @@
+import { Move } from '../../../jscaip/Move';
+import { LegalityStatus } from 'src/app/jscaip/LegalityStatus';
+import { Component } from '@angular/core';
+import { RectangularGameState } from 'src/app/jscaip/RectangularGameState';
+import { AbstractGameComponent } from '../abstract-game-component/AbstractGameComponent';
+import { Table } from 'src/app/utils/ArrayUtils';
+
+/* All method are to be implemented by the Concretes Game Component
```

Comment:
juste pour bitcher : pourquoi des majuscules à des noms communs ? (Concrete Game Component) + component*s*

Reply:
haha, ça fait plus "on parle de classe" j'imagine :D

Reply:
du coup, `by the concretes GameComponent classes` par exemple ?